closed/custom/Main.gmk

```diff
@@ -119,3 +119,53 @@ ifneq (,$(HEALTHCENTER_JAR))
   # The content must be extracted before module-info can be compiled.
   ibm.healthcenter-java : ibm.healthcenter-copy
 endif # HEALTHCENTER_JAR
+
+ifeq (true,$(BUILD_JGSKIT))
+
+# Identify the desired JGSKIT target platform.
+JGSKIT_TARGET :=
+JGSKIT_MAKE := jgskit.mak
+JGSKIT_MAKE_PATH := $(OPENJCEPLUS_TOPDIR)/src/main/native/
+JCE_CLASSPATH := $(OPENJCEPLUS_TOPDIR)/src/main/java:$(TOPDIR)/src/java.base/share/classes
+JCE_CLASSPATH_WIN64 := "$(call MixedPath,$(OPENJCEPLUS_TOPDIR)/src/main/java)\;$(call MixedPath,$(TOPDIR)/src/java.base/share/classes)"
+
+ifeq ($(OPENJDK_TARGET_OS), aix)
+  JGSKIT_TARGET := ppc-aix64
+else ifeq ($(OPENJDK_TARGET_OS), linux)
+  ifeq ($(OPENJDK_TARGET_CPU), x86_64)
+    JGSKIT_TARGET := x86-linux64
+  else ifeq ($(OPENJDK_TARGET_CPU), ppc64le)
+    JGSKIT_TARGET := ppcle-linux64
+  endif
+else ifeq ($(OPENJDK_TARGET_OS), windows)
+  ifeq ($(OPENJDK_TARGET_CPU), x86_64)
+    JGSKIT_TARGET := win64
+    JGSKIT_MAKE := jgskit.win64.mak
+    JCE_CLASSPATH := $(JCE_CLASSPATH_WIN64)
+    GSKIT_HOME := $(call MixedPath,$(GSKIT_HOME))
+    BOOT_JDK := $(call MixedPath,$(BOOT_JDK))
+  endif
+endif # OPENJDK_TARGET_OS
+
+ifeq (,$(JGSKIT_TARGET))
+  $(error Unsupported platform $(OPENJDK_TARGET_OS)-$(OPENJDK_TARGET_CPU))
+endif # JGSKIT_TARGET
+
+.PHONY : clean-openjceplus openjceplus-libs
+
+clean : clean-openjceplus
+
+clean-openjceplus :
+	@$(ECHO) Cleaning OpenJCEPlus native
+	@$(RM) -f $(addprefix $(JGSKIT_MAKE_PATH), com_ibm_crypto_plus_provider_icc_NativeInterface.h com_ibm_crypto_plus_provider_icc_FastJNIBuffer.h libjgskit.x)
```

Comment:
Please split this and sort the names:
```
	@$(RM) -f \
		$(addprefix $(JGSKIT_MAKE_PATH)/, \
			com_ibm_crypto_plus_provider_icc_FastJNIBuffer.h \
			com_ibm_crypto_plus_provider_icc_NativeInterface.h \
			libjgskit.x)
```
Note that `JGSKIT_MAKE_PATH` does not end with a `/`.

Reply:
When building the OpenJCEPlus with Semeru OpenJDK, the generated class and native files are in the `/OpenJCEPlus/target/` directory. So, do you think we still need these clean codes? I removed those OpenJCEPlus clean codes for now.